go/libraries/doltcore/sqle/enginetest/dolt_harness.go

```diff
@@ -120,7 +120,7 @@ func (d *doltHarness) NewTableAsOf(db sql.VersionedDatabase, name string, schema
 // branch with the given name.
 func (d *doltHarness) SnapshotTable(db sql.VersionedDatabase, name string, asOf interface{}) error {
 	ddb := db.(sqle.Database)
-	e := enginetest.NewEngineWithDbs(d.t, d.Parallelism(), []sql.Database{db}, nil)
+	e := enginetest.NewEngineWithDbs(d.t, d, []sql.Database{db}, nil)
```

Comment:
we don't need an index driver?

Reply:
Nope, got rid of that about a month ago. Tables provide their own indexes now, no driver needed.